s3cortxutils/s3confstore/ut/s3confstore_unit_tests.py

```diff
@@ -161,6 +161,7 @@ def test_mock_save(self, mock_load_return, mock_get_return, mock_set_return, moc
     result_data = s3confstore.get_config('bridge')
     self.assertTrue(result_data == mock_get_return.return_value)
 
+  @unittest.skip("Skipping as no need to test validation")
```

Comment:
Don't remove teste - lets skip the validation if URL is consul://

Reply:
fixed